.github/workflows/build.yml

```diff
@@ -56,11 +56,11 @@ jobs:
           opt_level: avx2
   linux-x86_64-AVX512-cmake:
     name: Linux x86_64 AVX512 (cmake)
-    if: false # TODO: enable when GitHub Actions adds AVX-512 hosts
-    needs: linux-x86_64-cmake
```

Comment:
Could we please gate avx-512 behind the plain vanilla build like all others?

`needs: linux-x86_64-cmake`